ztp/resource-generator/README.md

```diff
@@ -0,0 +1,88 @@
+# README #
+
+This directory provides RAN site ArgoCD gitops pipeline container image definitions
```

Comment:
I think its better to explain the resource-generator without tying it to ArgoCD. Its main purpose to use policyGen in container image to generate CRs.

Reply:
The concept of pre-sync and post-sync resource hooks is unique to the ArgoCD pipeline, so I think it's more than just packing the policy generator into a container. Maybe the resource-generator folder should be renamed?